test/unit/vl-select.test.html

```diff
@@ -399,8 +408,48 @@
           done();
         });
       });
+
+      test('de select placeholders bevatten default waarden', (done) => {
+        const select = fixture('vl-select-auto-dress-fixture');
+        awaitUntil(() => select._dressed).then(() => {
+          assert.equal(vl.i18n.i18n['select.no_more_options'], 'Geen resterende opties gevonden');
+          assert.equal(vl.i18n.i18n['select.placeholder_value'], 'Selecteer item');
+          assert.equal(vl.i18n.i18n['select.no_results'], 'Geen resultaten gevonden');
+          assert.equal(vl.i18n.i18n['select.search_placeholder_value'], 'Zoek item');
+          done();
+        });
+      });
+
+      test('kan waarde van search placeholder wijzigen via attribuut', (done) => {
+        const select = fixture('vl-select-auto-dress-fixture');
+        const nieuweSearchPlaceholderText = 'Dit is een nieuwe tekst';
+        awaitUntil(() => select._dressed).then(() => {
+          select.setAttribute('data-vl-search-placeholder', nieuweSearchPlaceholderText);
+          assert.equal(vl.i18n.i18n['select.search_placeholder_value'], nieuweSearchPlaceholderText);
+          done();
+        });
+      });
+
+      test('kan waarde van search no more options wijzigen via attribuut', (done) => {
+        const select = fixture('vl-select-auto-dress-fixture');
+        const nieuweSearchNoMoreOptionsText = 'Dit is een nieuwe tekst';
+        awaitUntil(() => select._dressed).then(() => {
+          select.setAttribute('data-vl-search-no-results-text', nieuweSearchNoMoreOptionsText);
+          assert.equal(vl.i18n.i18n['select.no_more_options'], nieuweSearchNoMoreOptionsText);
```

Comment:
Als die vertalingen op een object op de global scope (vl) worden bijgehouden, hoe kan dan elke component zijn eigen message hebben?

Reply:
Geen idee. Het is me voor sommige teksten zelfs niet gelukt om die in een applicatie zichtbaar te maken. Verder is er ook nog een attribuut (data-vl-select-search-empty-text) op niveau van de AIV select component waarmee je meegeeft welke tekst moet getoond worden als er geen resultaten gevonden worden.

Reply:
Ik zou dan precies wel zien da ge eerst wel een idee hebt ...

Reply:
Voor het data-vl-search-placeholder attribuut werd dit in dit ticket wel opgelost (in de dress functie), door te kijken of dat attribuut op de component zat en de vertaling daarvoor dan aan te passen. Als die niet bestond wordt de default waarde aan de vertaling gegeven net voor de component 'gedressed' wordt.
Dit kan eventueel ook voor de andere 'vertaling-keys', maar dit heb ik niet gedaan omdat het heel onduidelijk was of deze al dan niet gebruikt werden en wanneer ze getoond worden (zie ook dat extra attribuut door AIV aangeleverd in mijn comment hierboven)

Reply:
Ja maar we zitten daar dus een globale variabele aan te passen die dan "toevallig op het juiste moment de juiste waarde heeft", en bij de volgende select weer terug een andere waarde.

Reply:
Wat er zou moeten gebeuren is voor de dress de globale variabelen goed zetten zodat de creatie nadien de correcte waarden gebruikt. Wanneer de gebruiker geen teksten wijzigt, moet overal de default gereset worden om te voorkomen dat de (door bijvoorbeeld een andere component) laatst gedefinieerde globale select i18n teksten gebruikt worden. In deze PR werd dit blijkbaar uitsluitend voor de `data-vl-search-placeholder` gedaan, maar dit kan best voor elke select i18n variabele gebeuren.

Reply:
Wat er "beter" zou moeten gebeuren is toch dat er daar niet met globale variabelen wordt gewerkt in die lib-code?

Reply:
Akkoord, het staat ons altijd vrij om een beter voorstel te doen en dit centraal voor te stellen ter adoptie.